api/services/taxonomyService.ts

```diff
@@ -0,0 +1,27 @@
+import { makeSCOSRequest } from './scosService';
+import getLogger from '../utils/logger';
+import TaxonomyDto from '../models/dto/taxonomyDto';
+import ScosSubcategoryDto from '../models/scosApi/scosSubcategoryDto';
+import ScosCategoryDto from '../models/scosApi/scosCategoryDto';
+
+const { CATEGORIES_TABLE } = process.env;
+
+const log = getLogger('taxonomyService');
+
+async function getSubcategories(id: string): Promise<TaxonomyDto[]> {
+  log.debug(`Getting subcategories of category id ${id}`);
+
+  const query = `SELECT subcat FROM ${CATEGORIES_TABLE} WHERE categoryid = '${id}'`;
+  const response: Pick<ScosCategoryDto, 'subcat'>[] = await makeSCOSRequest(query);
+
+  return response[0].subcat.map((subcat: ScosSubcategoryDto): TaxonomyDto => ({
```

Comment:
Does this schema match what the frontend is expecting? I don't remember using `description` for the name but I could be wrong

Reply:
yup, I checked against the frontend :) I wasn't able to 100% test these changes alongside the UI, but I will definitely do that when more endpoints are in place.